projects/packages/videopress/src/client/block-editor/blocks/video/components/player/index.native.tsx

```diff
@@ -44,6 +68,17 @@ export default function Player( {
 
 			{ ! isRequestingEmbedPreview && <SandBox html={ html } /> }
 			{ ! html && <Text>{ __( 'Loading…', 'jetpack-videopress-pkg' ) }</Text> }
+
+			{ isSelected && (
```

Comment:
Based on the PR's description:
> - The caption is visible when the block is selected.
> - The caption is always displayed regardless of the block settings *

It's not clear whether the caption should only be displayed when is selected. I checked other blocks and the caption, if it has a value, is always displayed. Hence, I think we don't need this condition.